src/ChainSafe.Gaming/Web3/Core/IProjectConfig.cs

```diff
@@ -9,5 +9,7 @@ public interface IProjectConfig
         /// The project id issued by ChainSafe. Follow https://docs.gaming.chainsafe.io to learn more.
         /// </summary>
         public string ProjectId { get; }
+
+        public bool AnalyticsOptOut { get; }
```

Comment:
Set it here to equal True. AnalyticsOptOut => true;

Reply:
done